DQMOffline/Trigger/src/EgHLTMonElemFuncs.cc

```diff
@@ -95,6 +96,99 @@ void MonElemFuncs::initStdPhoHists(std::vector<MonElemManagerBase<OffPho>*>& his
 							     bins.etaVsPhi.nrY,bins.etaVsPhi.yMin,bins.etaVsPhi.yMax,
 							     &OffPho::hltEta,&OffPho::hltPhi));
 }
+
+
+void MonElemFuncs::initStdEleHistsHEP(std::vector<MonElemManagerBase<OffEle>*>& histVec,const std::string& filterName,const std::string& baseName,const BinData& bins)
```

Comment:
honestly I didn't think about it ... but there are other histograms and efficiency plots which are not needed for new folder(e.g. I removed di object plots from MonElemFuncs). The efficiency plots are currenlty "off" but in future those could be turned "on". So I find it simpler(adding those if() ) to do this way.